V10-BreakingChanges.md

```diff
@@ -1,5 +1,58 @@
-# A collection of updates that change the behaviour
+# A collection of updates that change the behavior
+
```

Comment:
We need to document that Mermaid is now ESM only.

Ideally we should show some example usage too.